dev/com.ibm.websphere.appserver.webProfile-8.0/com.ibm.websphere.appserver.webProfile-8.0.feature

```diff
@@ -0,0 +1,28 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.webProfile-8.0
+visibility=public
+IBM-App-ForceRestart: install, uninstall
+IBM-ShortName: webProfile-8.0
+Subsystem-Version: 8.0.0
+Subsystem-Name: Java EE Web Profile 8.0
+# TODO-EE8: Need to add appSecurity-3.0 once bug w/ CDI 2.0 is resolved
```

Comment:
appSecurity-3.0 will bring in jaspic-1.1, but we should probably list it explicitly...